src/basic/random-util.c

```diff
@@ -42,11 +42,12 @@
 #include "random-util.h"
 #include "time-util.h"
 
-int dev_urandom(void *p, size_t n) {
+int dev_urandom(void *p, size_t n, bool pseudorandom_ok) {
```

Comment:
i think i'd turn the flag around: "high_quality_required" or so. Otherwise it's a "hidden negation", no?